lib/components/molecules/QuoteMedia/types/index.js

```diff
@@ -4,8 +4,6 @@ import type { Node } from 'react';
 export type Props = {
   className?: string,
```

Comment:
every option prop should have a default value

Reply:
fixed